### ISSUE TITLE
I can use schemaZen with a Local Compact Database?

### ISSUE THREAD

**Issue by KarloDev** [sethreno/schemazen#35]
Hi, I want to get the script of a Local 4.0 Compact SQL database but I dont know If schemazen might be able to achieve this, It would be great. Thanks.

**Comment by keith-hall** [comment]
Hi @KarloDev, I have done some investigation and the answer is that no, currently SchemaZen doesn't support SQL CE.  Accessing a compact SQL database [requires the use of a separate assembly, System.Data.SqlServerCe.dll](https://msdn.microsoft.com/en-us/library/system.data.sqlserverce.sqlceconnection%28v=vs.100%29.aspx). [This assembly is installed separately from the .NET framework](http://stackoverflow.com/questions/11714480/i-dont-have-sqlserverce-assembly), and exists in a Program Files subfolder, Microsoft SQL Server Compact Edition\v3.5\Private. I have tried referencing this, but [get an error](https://support.microsoft.com/en-us/kb/974247). I wonder whether it might end up being too much hassle for end-users to get it working...

**Comment by sethreno** [comment]
I agree with @keith-hall on this, I wouldn't want to add a dependency on SqlServerCe.dll. However, I would be okay if an attempt were made to locate the dll and load it dynamically when scripting a SQL CE database. I'll mark this as up for grabs in case someone is interested in implementing it.
